### ISSUE TITLE
Booleans doesn't work

### ISSUE THREAD

**Issue by Marfusios** [zoomsphere/ngx-store#22]
Lib version: `1.1.10`
Angular version: `4.4.3`

Example code: 
```
import {Injectable} from '@angular/core';
import {LocalStorage} from 'ngx-store';

@Injectable()
export class KoFeatureTogglesService {

  @LocalStorage('ko_feature_overlay_panels')
  public showOverlayPanels: boolean = true;

  @LocalStorage('ko_feature_developer_panels')
  public showDeveloperPanels: boolean = true;

  constructor() {}
}
```

Booleans are stored correctly into local storage: 
![image](https://user-images.githubusercontent.com/3494837/32161880-6bf98c4c-bd57-11e7-8d0f-6f550271f606.png)

But they are not loaded on start (default value is used). 
Log: 
![image](https://user-images.githubusercontent.com/3494837/32162147-3d67c88e-bd58-11e7-9d1b-89d8fe1f5ee3.png)
![image](https://user-images.githubusercontent.com/3494837/32162208-6d5e1372-bd58-11e7-93ea-a3f642fa6190.png)

Did anyone experience this? Did I do something wrong?

**Comment by DanielKucal** [comment]
Hi @Marfusios, thank you for this report. The issue is now fixed in `v1.1.11`.

**Comment by Marfusios** [comment]
Thank you for the fix, works
